crates/eframe/src/lib.rs

```diff
@@ -144,6 +144,15 @@
 #![warn(missing_docs)] // let's keep eframe well-documented
 #![allow(clippy::needless_doctest_main)]
 
+// Limitation imposed by `accesskit_winit`:
+// https://github.com/AccessKit/accesskit/tree/accesskit-v0.18.0/platforms/winit#android-activity-compatibility`
+#[cfg(all(
+    target_os = "android",
+    feature = "accesskit",
+    feature = "android-native-activity"
+))]
+compile_error!("`accesskit` feature is only available with `android-game-activity`");
+
```

Comment:
I appreciate improving the error message. Without this, what would the user see?

I'm a bit worried that maybe accesskit_winit removed this limitation, and then we forget to update this code

Reply:
Nothing. Then it panics at runtime on `unwrap`:

https://github.com/AccessKit/accesskit/blob/341a11bca2c8a29682c11ddcfe91fa58776ea11d/platforms/winit/src/platform_impl/android.rs#L35